.github/workflows/linux.yml

```diff
@@ -16,476 +16,490 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  PYTHON_VERSION: '3.9'
-  OV_BRANCH: 0080d90974ca84f9a6d359da3388a2a18a93b753
-  OV_TARBALL: ''
+  PYTHON_VERSION: '3.11'
+  OV_BRANCH: ${{ github.base_ref || github.event.merge_group.base_ref || github.ref }}
+  CMAKE_CXX_COMPILER_LAUNCHER: sccache
+  CMAKE_C_COMPILER_LAUNCHER: sccache
+  SCCACHE_IGNORE_SERVER_IO_ERROR: 1
+  SCCACHE_SERVER_PORT: 35555
+  SCCACHE_CACHE_SIZE: 30G
+  SCCACHE_AZURE_KEY_PREFIX: genai/ubuntu/22_04/x64
+  GENAI_ARCHIVE_NAME: genai.tar.gz
+  GENAI_SAMPLES_NAME: genai_samples.tar.gz
+  OV_SAMPLES_NAME: ov_samples.tar.gz
 
 jobs:
   openvino_download:
-    name: Download OpenVINO package
+    name: Download Prebuilt OpenVINO
     outputs:
       status: ${{ steps.openvino_download.outcome }}
+      ov_artifact_name: ${{ steps.openvino_download.outputs.ov_artifact_name }}
+      ov_wheel_source: ${{ steps.openvino_download.outputs.ov_wheel_source }}
+      ov_version: ${{ steps.openvino_download.outputs.ov_version }}
+      docker_tag: ${{ steps.get_docker_tag.outputs.docker_tag }}
     timeout-minutes: 10
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04
+    runs-on: aks-linux-4-cores-16gb
```

Comment:
because we faced with an issues with the aks-linux-2-cores-8gb, because it is used by statistic jobs.  We would need it split runners scopes first. @ababushk fyi